src/widgets/withOnePress.js

```diff
@@ -9,10 +9,13 @@ import React from 'react';
  */
 export const withOnePress = WrappedComponent =>
   class extends React.Component {
-    state = { hasBeenPressed: false };
-
     static propTypes = WrappedComponent.propTypes;
 
+    constructor(props) {
```

Comment:
For
`  12:5  error  State initialization should be in a constructor        react/state-in-constructor`